app/views/layouts/application.html.haml

```diff
@@ -43,6 +43,7 @@
       = render_breadcrumbs(builder: govuk_breadcrumb_builder)
       %main.govuk-main-wrapper#main-content{ role: "main" }
         = render partial: 'layouts/flashes' unless flash.empty?
+        = content_for :error_summary
```

Comment:
This is the important part!
Ehe error summary must be the first thing in the page content, according with the GDS guide: https://design-system.service.gov.uk/components/error-summary/